### PR TITLE
Media type & double underscore

### LINKED ISSUE #64: Update names of submitters in part v
5 people are listed - Jim is no longer at NOAA, need to check with Aleksander if he would like to continue

### LINKED ISSUE #107: Preferred abbreviation for 1-d, 2-d
@marqh and others. This is a very minor point, but I worked through it with Carl Reed in other OGC documents. I suggest that '1-d', '2-d', etc are replaced by '1D' and '2D' etc. Reasons:

1. The hyphen makes the strings not very searchable/findable;
2.  '2d' and '3d' were often used in old documents to mean second and third (and Google has scanned many of them);
3. The upper case 'D' stands out as an abbreviation.

HTH

### LINKED ISSUE #108: justify the choice of '__' as a namespace separator
at the OGC SWG meeting on 2nd December 2020 the choice of the double underscore character for namespace separation within netCDF CDL & encodings was discussed

this has also been highlighted in the open issue:
#50 

further justification on this implementation choice is requested within the informative sections of the standard document to ensure that the justification can be confirmed by the SWG as part of the proposed vote

### PR DESCRIPTION
**Pull request by marqh**
address new media type definition

justify double underscore

fixes #107 
fixes #64 
fixes #108

### CODE REVIEW

**Comment by marqh** [comment]
@jyucsiro @adamml  is this ready to merge now, do you think?

or are the outstanding changes required?

File: standard/clause_6_informative_text.adoc [hunk 1]
```diff
@@ -311,6 +311,19 @@ The definition of multiple prefix resources within a single file is invalid in t
 netCDF-LD implementations may choose to combine prefix collections in invalid cases, but no precedence is implied, and prefix conflict is unresolved. 
 netCDF-LD implementations may treat this condition as a warning condition and as a validation error.
 
+====== The Double Underscore Syntax
+
+This standard introduces a special text syntax, a double underscore '__' as a separator between the prefix and suffix.
+
+The choice of special character has been chosen as a compromise of various factors.  RDF uses a ':' (colon) as a separator between prefix and suffix, but the netCDF CDL specification uses the colon as a variable attribute separator.  As such, the use of a colon as a separator requires escape characters to be used.
+
+Older versions of the netCDF libraries precluded the use of colon at all, the ability to use the escape character was introduced, but some use of double underscore was already in use in ad-hoc file encodings.  Maintaining backwards compatibility is deemed a seful facet of this syntax.
```
Comment (jyucsiro) [inline standard/clause_6_informative_text.adoc:320]: seful -> useful?

**Review by jyucsiro — COMMENTED** [review]
Just a minor typo. Otherwise - good to go

**Comment by adamml** [comment]
Looks good to me once the typo Jonathan spotted is fixed

File: standard/clause_6_informative_text.adoc (hunk 1 above)
Comment (marqh) [reply standard/clause_6_informative_text.adoc]: resolved

**Review by jyucsiro — APPROVED** [review]
looks good!

**Pull request merged**